### ISSUE TITLE
When multiple indexablees, only the first indexable is displayed

### ISSUE THREAD

**Issue by steve02081504** [TypeStrong/typedoc#2470]
## Search terms

Indexable muti

## Expected Behavior

https://github.com/ukatech/jsstp-lib/blob/2cb88bcf51053b4c6b16a8cdaa613932e5779c7d/dist/en/jsstp.d.ts#L485L505
```
/**
 * Matches event names to generate a simple invoker
 * @group Index reflections
 * @example
 * let data=await jsstp.OnTest(123,"abc");
 */
[key: `On${string}`]: simple_event_caller;
/**
 * Matches event names to generate a simple invoker
 * @group Index reflections
 * @example
 * let data=await jsstp.GetNames();
 */
[key: `Get${string}`]: simple_list_command_caller;
/**
 * Matches event names to generate a simple invoker
 * @group Index reflections
 * @example
 * let data=await jsstp.SetCookie("abc","def");
 */
[key: `Set${string}`]: simple_command_caller;
```
The document should show all 3 indexables.

## Actual Behavior

https://ukatech.github.io/jsstp-lib/doc/EN/classes/jsstp_t.html
just first one

## Steps to reproduce the bug

```ts
class cla {
[key: `1${string}`]: number;
[key: `2${string}`]: string;
}
```
maybe this

## Environment

-   Typedoc version: `^0.25.3`
-   TypeScript version: idk
-   Node.js version: `latest`
-   OS: `ubuntu-latest`
https://github.com/ukatech/jsstp-lib/blob/master/.github/workflows/static-page.yml

**Comment by Gerrit0** [comment]
Hey, someone finally noticed, only took ~9 years!

**Comment by Gerrit0** [comment]
Fixed with TypeDoc 0.26, which is releasing 2024/06/21
